components/View/index.jsx

```diff
@@ -68,10 +71,22 @@ class View extends Component {
     this.element = null;
 
     this.state = {
-      keyboardHeight: 0,
+      noScroll: false,
     };
+  }
+
+  /**
+   * Adds keyboardWillChangeListener.
+   */
+  bindKeyboardChange() {
+    event.addCallback(EVENT_KEYBOARD_WILL_CHANGE, this.handleKeyboard)
```

Comment:
You don't register for `EVENT_KEYBOARD_WILL_CHANGE`. So when `noScrollOnKeyboard` is used in a view before the `SearchField` component was visible once, it will not work.

Reply:
Ups. My bad. Fixed.